lib/dash/content_protection.js

```diff
@@ -250,6 +250,179 @@ shaka.dash.ContentProtection.parseFromRepresentation = function(
   return repContext.defaultKeyId || context.defaultKeyId;
 };
 
+/**
+ * Gets a Widevine license URL from a content protection element
```

Comment:
Unless I am mistaken, 'ms:laurl' is a Microsoft specific element.  I don't believe Widevine has a standard way of embedding URLs in the manifest.